### PR TITLE
Add poor-man RPi detection on the FreeBSD

### PR DESCRIPTION
**Pull request by samm-git**
Tested on RPI 1b and RPI4.

### CODE REVIEW

**Comment by samm-git** [comment]
P.S. To make it _reall_ working we should use ofwdump tool. 

E.g.
```
# ofwdump -P model -S /
Raspberry Pi 4 Model B Rev 1.2
```
or 
```
# ofwdump -P model -S /
Raspberry Pi Model B Rev 2
```

File: hw-probe.pl [hunk 1]
```diff
@@ -3753,6 +3753,18 @@ ()
         }
     }
     
+    if(not $Sys{"Type"} and not $Sys{"Model"})
+    {
+        for my $str ($Kldstat_v) {
```
Comment (linuxhw) [inline hw-probe.pl:3758]: May be rewrite this like the following?

    if($Kldstat_v=~/bcm2835_/)
    {
        $Sys{"Type"} = "system on chip";
        $Sys{"Model"} = "Raspberry Pi";
    }

Comment (samm-git) [reply hw-probe.pl:3758]: yes, was not touching perl a long time :)

```
System: freebsd-13.0-BETA4
Arch:   arm64
Kernel: 13.0-BETA4
Model:  Raspberry Pi
Type:   system on chip
Hwaddr: 0fed5e133edb751c1b8e0b2f86740ab3
```

Comment (linuxhw) [reply hw-probe.pl:3758]: Please update your commit accordingly.

Comment (samm-git) [reply hw-probe.pl]: Done )

**Comment by linuxhw** [comment]
Thanks!

**Comment by linuxhw** [comment]
Next time, try to join all your commits in one PR into one cumulative commit.

**Comment by linuxhw** [comment]
Do we have a probe of it in the database?

**Pull request merged**